pipelines/docker-build-oci-ta.yaml

```
@@ -193,12 +189,15 @@ spec:
     - name: BUILD_ARGS
       value:
       - $(params.build-args[*])
+      - SENTRY_RELEASE=$(tasks.clone-repository.results.commit)
     - name: BUILD_ARGS_FILE
       value: $(params.build-args-file)
     - name: SOURCE_ARTIFACT
       value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
     - name: CACHI2_ARTIFACT
       value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+    - name: ADDITIONAL_SECRET
+      value: sentry-auth
```

What happens to the pipelines that depend on `pipelines/docker-build-oci-ta.yaml` if `sentry-auth` does not exist?

---

I dont expect this to affect apps that do not have the sentry-auth variable. This just allows the variable to be passed, if nothing is passed, the plugin will never get initialized but thats all that should happen

---

`ADDITIONAL_SECRET` is a param from [buildah](https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-oci-ta/0.4/README.md). If it's present here, buildah will likely try to use the secret and maybe fail when it's missing. That's why I asked the question above.

---

In conjunction with the docker PR: https://github.com/RedHatInsights/insights-frontend-builder-common/pull/256 

We pass ADDITIONAL_SECRET: sentry-auth to the buildah task. That mount is optional in the task, and the Dockerfile’s line uses RUN --mount=type=secret,id=sentry-auth,required=false, so the build step won’t fail if the secret isn’t present.

Inside the RUN script we check TOKEN_FILE="/run/secrets/sentry-auth/${APP_NAME}" and only export ENABLE_SENTRY=true + SENTRY_AUTH_TOKEN=... if the file exists and is non-empty. Otherwise you print the “no token… skipping upload” message and continue.

Because we're not setting ENABLE_SENTRY from the pipeline anymore, apps without a token keep ENABLE_SENTRY=false, so the Sentry plugin doesn’t run and no upload is attempted.